app/helpers/address_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module AddressHelper
+  def postcode_lookup(postcode)
+    postcode_url = "#{ENV['API_URL']}?postcode=#{postcode}&key=#{ENV['API_KEY']}"
```

Comment:
One thing we discovered was that the default response excludes flats that do not have their own street address.  They can be included by adding `dataset=LPI` to the query string.

Reply:
Thanks @brucebolt - I've update as per your suggestion.